vars/copyContainer.groovy

```diff
@@ -10,34 +10,67 @@
  * Copies a container from one docker registry to another
  *
  * @param args A map of the following parameters
+ * @param args.recursiveCopy Copy all the tags of a sourceImage to destinationImage if 'true' and <IMAGE_TAG> is ignored in sourceImage/destinationImage, default to 'false'
  * @param args.sourceImage The Source Image name and tag <IMAGE_NAME>:<IMAGE_TAG> Eg: opensearch:1.3.2
```

Comment:
could you add which parameters are required and which are optional?

Reply:
Added now.